packages/thumbnail/README.md

```diff
@@ -26,69 +26,167 @@ import { Thumbnail } from '@spectrum-web-components/thumbnail';
+<sp-tab-panel value="100">
 
 ```html
-<sp-thumbnail size="s">
+<sp-thumbnail size="100">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
-<sp-tab value="m">Medium</sp-tab>
-<sp-tab-panel value="m">
+<sp-tab value="200">200</sp-tab>
+<sp-tab-panel value="200">
 
 ```html
-<sp-thumbnail size="m">
+<sp-thumbnail size="200">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
-<sp-tab value="l">Large</sp-tab>
-<sp-tab-panel value="l">
+<sp-tab value="300">300</sp-tab>
+<sp-tab-panel value="300">
 
 ```html
-<sp-thumbnail size="l">
+<sp-thumbnail size="300">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="400">400</sp-tab>
+<sp-tab-panel value="400">
+
+```html
+<sp-thumbnail size="400">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="500">500</sp-tab>
+<sp-tab-panel value="500">
+
+```html
+<sp-thumbnail size="500" style="display: flex;">
+    <div class="image-wrapper">
+        <img
+            class="image"
+            style="max-block-size: 100%;
+                max-inline-size: 100%;
+                position: relative;
+                z-index: 1;"
+            src="https://place.dog/100/100"
+            alt="Demo Image"
+        />
+    </div>
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="600">600</sp-tab>
+<sp-tab-panel value="600">
+
+```html
+<sp-thumbnail size="600">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="700">700</sp-tab>
+<sp-tab-panel value="700">
+
+```html
+<sp-thumbnail size="700">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="800">800</sp-tab>
+<sp-tab-panel value="800">
+
+```html
+<sp-thumbnail size="800">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="900">900</sp-tab>
+<sp-tab-panel value="900">
+
+```html
+<sp-thumbnail size="900">
+    <img src="https://place.dog/100/100" alt="Demo Image" />
+</sp-thumbnail>
+```
+
+</sp-tab-panel>
+<sp-tab value="1000">1000</sp-tab>
+<sp-tab-panel value="1000">
+
+```html
+<sp-thumbnail size="1000">
     <img src="https://place.dog/100/100" alt="Demo Image" />
 </sp-thumbnail>
 ```
 
 </sp-tab-panel>
 </sp-tabs>
 
-## Focused or selected
+## Focused
 
-When `focused` or `selected` the `sp-thumbnail` element will be displayed as follows:
+When `focused` the `sp-thumbnail` element will be displayed as follows:
+
+```html
+<sp-thumbnail focused style="display: flex;">
+    <div class="image-wrapper">
+        <img
+            class="image"
+            style="max-block-size: 100%;
+                max-inline-size: 100%;
+                position: relative;
+                z-index: 1;"
```

Comment:
same comments as above this should work without needing the inline styling for the `<img>`  image or  `display: flex;`  on` <sp-thumbnail>`

Reply:
got this fixed!